packages/api/src/getPublicKey/getPublicKey.ts

```diff
@@ -1,29 +1,33 @@
-import { GEM_WALLET, RequestPublicKeyMessage } from '@gemwallet/constants';
+import {
+  GEM_WALLET,
+  PublicKeyResponsePayload,
+  RequestGetPublicKeyMessage
+} from '@gemwallet/constants';
 
 import { sendMessageToContentScript } from '../helpers/extensionMessaging';
 import { getFavicon } from '../helpers/getFavicon';
 
-export const getPublicKey = async () => {
+export const getPublicKey = async (): Promise<PublicKeyResponsePayload> => {
```

Comment:
```suggestion
export const getPublicKey = async (): Promise<GetPublicKeyResponsePayload> => {
```
Please well export the type: `GetPublicKeyResponsePayload` to the API